src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -256,7 +256,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       Condition="'$(IncludeBuildOutput)' == 'true'"
       Projects="$(MSBuildProjectFullPath)"
       Targets="_GetBuildOutputFilesWithTfm"
-      Properties="TargetFramework=%(_TargetFrameworks.Identity);">
+      Properties="TargetFramework=%(_TargetFrameworks.Identity);
+                  ExcludeRestorePackageImports=true;">
```

Comment:
Adding global properties to an MSBuild task invocation like this will cause MSBuild to re-build the transitive closure of targets for the target(s) you're asking to build. If `_GetBuildOutputFilesWithTfm` depends on other targets (or other targets have `BeforeTargets` or `AfterTargets` set on this one) I'm concerned that you'll end up double-building them as a result of this. And as MSBuild targets often have non-idempotent side effects and dramatic impact on build perf, we should strive to keep the global properties the same everywhere we can. 
Was the set of global properties you had before this change identical to those used for inner-build invocations from the dotnet SDK targets?